audit/gdcd/README.md

```diff
@@ -58,7 +58,10 @@ For each docs page:
 
 ### Prerequisites
 
-Before you begin, contact the Developer Docs team for the required connection details and access. 
+Before you begin, contact the Developer Docs team for the required
```

Comment:
My only question here is should we be capturing these details here in the README, or in an internal Wiki? i.e. if we're the only folks who can set up the `gdcdUser` and we are giving other people connection details, this seems like an internal implementation detail for _us_, not for someone who might want to run this and needs to have us create the user for them.

Reply:
Wiki updated 👍